### PR TITLE
fix(#1125): Fix scenario with OIDC and remote, authenticated Jolokia …

### PR DESCRIPTION
**Pull request by grgrzybek**
…agent

### CODE REVIEW

File: packages/hawtio/src/plugins/auth/oidc/oidc-service.ts [hunk 1]
```diff
@@ -195,9 +214,11 @@ export class OidcService implements IOidcService {
...
       // authorizationUrl.searchParams.set('login_hint', 'hawtio-viewer@fuseqe.onmicrosoft.com')
       // authorizationUrl.searchParams.set('hsu', '1')
-      if (config.prompt) {
-        authorizationUrl.searchParams.set('prompt', config.prompt)
-      }
+      // it's better to stick to default, as 'prompt=login' always
+      // shows login screen...
+      // if (config.prompt) {
+      //   authorizationUrl.searchParams.set('prompt', config.prompt)
+      // }
```
Comment (tadayosi) [inline packages/hawtio/src/plugins/auth/oidc/oidc-service.ts:221]: too many comment outs here and there. maybe we can clean them up now.

**Review by tadayosi — APPROVED** [review]

File: packages/hawtio/src/plugins/auth/keycloak/keycloak-service.ts [hunk 2]
```diff
@@ -117,6 +117,11 @@ class KeycloakService implements IKeycloakService {
     const initOptions: KeycloakInitOptions = {
       onLoad: 'login-required',
       pkceMethod,
+      // required for Keycloak 23
+      // see: https://github.com/keycloak/keycloak/issues/26651
+      // and we can't switch to Keycloak 25
+      // see: https://github.com/keycloak/keycloak/issues/27624
+      useNonce: false,
```
Comment (tadayosi) [inline packages/hawtio/src/plugins/auth/keycloak/keycloak-service.ts:124]: so should we pin the version of keycloak.js to 23.x?  related: #1133

Comment (grgrzybek) [reply packages/hawtio/src/plugins/auth/keycloak/keycloak-service.ts:124]: I'm not sure what to do yet (will get back to it later), so I only left the dev notes - I hope you don't mind ;)

File: packages/hawtio/src/plugins/auth/oidc/oidc-service.ts (hunk 1 above)
Comment (grgrzybek) [reply packages/hawtio/src/plugins/auth/oidc/oidc-service.ts]: I've removed the comment ;) Thanks for spotting!

File: packages/hawtio/src/plugins/auth/keycloak/keycloak-service.ts (hunk 2 above)
Comment (grgrzybek) [reply packages/hawtio/src/plugins/auth/keycloak/keycloak-service.ts:124]: @tadayosi are you ok with this explanation? :)

Comment (tadayosi) [reply packages/hawtio/src/plugins/auth/keycloak/keycloak-service.ts:124]: It's not a change request but a pure question :-)  I wonder what we should do for #1133. Feel free to resolve it.

**Pull request merged**